### PR TITLE
Unified token creation flow

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/10243

## Description of Changes
We had 2 token creation flows, a quick one with a single step and another one with 3 individual steps. To reduce friction, the user flow with more steps is now removed to match the quick token launch flow.

## "How We Fixed It"
N/A

## Test Plan
- Click on the "Launch Token" button from the sidebar
- Attempt to launch a token, and verify the steps/flow is same to the launch token flow from `/communities` => `Launch token` cta
- Verify no regressions occured

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

File: libs/model/src/services/openai/generateTokenIdea.ts
Comment (mzparacha) [inline libs/model/src/services/openai/generateTokenIdea.ts:1]: had to revert this change, as the images generated were >mb sizes and it didn't match up with create community APIs image size requirements.

**Review by masvelio — APPROVED** [review]

File: libs/model/src/services/openai/generateTokenIdea.ts
Comment (mzparacha) [reply libs/model/src/services/openai/generateTokenIdea.ts:1]: it doesn't seem to support image size < 1024x1024 px, and those were in mbs so I reverted it

**Review by Israellund — APPROVED** [review]
